build.cake

```diff
@@ -377,6 +376,19 @@ Task("AppVeyor")
 Task("Travis")
   .IsDependentOn("Run-Tests");
 
+Task("Azure-Linux")
+  .IsDependentOn("Run-Tests");
+
+Task("Azure-OSX")
+  .IsDependentOn("Run-Tests");
+
+Task("Azure-Windows")
+  .IsDependentOn("Package")
+  .IsDependentOn("Copy-Files-Impl")
+  .IsDependentOn("Zip-Files-Impl")
+  .IsDependentOn("Publish-MyGet-Impl")
```

Comment:
We should remove these two tasks from `Azure-Windows` since we'll handle them in the release pipeline eventually.